README.md

```diff
@@ -108,6 +108,7 @@ Application monitoring is powered by [Scout APM](https://scoutapm.com). This hel
 * Run `rails test` to run unit/integration tests.
 * Run `rails test:system` to run system tests, using `headless_chrome`.
 * Run `HEADFUL=1 rails test:system` to run system tests, using `headful_chrome`.
+* Run `bin/rails verify_mailer_previews` to verify mailer previews.
```

Comment:
@joemasilotti : I added this one here, because I was about to send you a comment about the RED CIs, after running the 3 above and getting GREEN on my local machine.

This one is failing on this PR and it works on `main` branch.
I will do some investigation.

Here is the error:
```
ActiveJob::SerializationError: Unable to serialize Business without an id. (Maybe you forgot to call save?)
/Users/hac/oss_dev/hac_forks/railsdevs.com/test/mailers/previews/admin_mailer_preview.rb:10:in `new_business'
/Users/hac/oss_dev/hac_forks/railsdevs.com/lib/tasks/verify_mailer_previews.rake:10:in `block (3 levels) in <top (required)>'

...
Caused by:
URI::GID::MissingModelIdError: Unable to create a Global ID for Business without a model id.
/Users/hac/oss_dev/hac_forks/railsdevs.com/test/mailers/previews/admin_mailer_preview.rb:10:in `new_business'
/Users/hac/oss_dev/hac_forks/railsdevs.com/lib/tasks/verify_mailer_previews.rake:10:in `block (3 levels) in <top (required)>'
```